sarracen/kernels/cubic_spline.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+
+from sarracen.kernels import BaseKernel
+
+
+class CubicSplineKernel(BaseKernel):
+    """
+    An implementation of the Cubic Spline kernel, in 1, 2, and 3 dimensions.
+    """
+    def __init__(self):
+        super().__init__(2.0, 2./3., 10./(7.*np.pi), 1./np.pi, self._weight)
+
```

Comment:
Here (and in other kernels), this becomes

def __init__(self, ndims)

and the normalization constant is chosen based on the dimensionality and that value is passed to super().